src/types.ts

```diff
@@ -73,6 +73,7 @@ export interface TaggedTextOptions {
   wrapEmoji?: boolean;
   errorHandler?: ErrorHandler;
   supressConsole?: boolean;
+  cacheCapacity?: number;
```

Comment:
Should this be based, for example, on the size of `this.text` rather than just arbitrarily decided by the user?